src/components/User/User.jsx

```diff
@@ -0,0 +1,15 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import './user.scss';
+
+export const User = ({ user }) => (
+  <div className="toDosList__item-person">{user.name}</div>
+);
+
+User.propTypes = {
+  user: PropTypes.objectOf(
+    PropTypes.shape({
+      name: PropTypes.string.isRequired,
+    }),
+  ).isRequired,
```

Comment:
no, do not use objectOf
```suggestion
  user: PropTypes.shape({
      name: PropTypes.string.isRequired,
    }).isRequired,
```